src/bootstrap/native.rs

```diff
@@ -94,6 +94,7 @@ pub fn llvm(build: &Build, target: &str) {
        .profile(profile)
        .define("LLVM_ENABLE_ASSERTIONS", assertions)
        .define("LLVM_TARGETS_TO_BUILD", llvm_targets)
+       .define("LLVM_EXPERIMENTAL_TARGETS_TO_BUILD", "WebAssembly")
```

Comment:
See how `LLVM_TARGETS_TO_BUILD` can be configured by `config.llvm_targets`? I think we better do similar by adding `config.llvm_experimental_targets` - it doesn't seem right to allow all the targets to be configured out except the experimental one. It should be reasonably obvious how to make that change, but it will also require updating [the config file example](https://github.com/rust-lang/rust/blob/master/src/bootstrap/config.toml.example#L54).